docs/advanced/cursors.rst

```diff
@@ -68,7 +68,7 @@ server (for instance when fetching new records or when moving using
 `~Cursor.scroll()`).
 
 Using a server-side cursor it is possible to process datasets larger than what
-would fit in the client memory. However for small queries they are less
+would fit in the clients memory. However for small queries they are less
```

Comment:
Why plural?

Reply:
This should be `client's` (possessive).
i.e. the memory belonging to the client.

Reply:
OK.